src/filtrex.js

```diff
@@ -210,8 +210,8 @@ function filtrexParser() {
                 ['argsList , e', code([1, ',', 3], true)],
             ],
             inSet: [
-                ['e', code(['o ==', 1], true)],
-                ['inSet , e', code([1, '|| o ==', 3], true)],
+                ['e', code(['!Array.isArray(o) ? o == ', 1, ': o.indexOf(', 1, ') >= 0'])],
```

Comment:
This code is checking if `o` is an array for every value of the array literal. That seems quite redundant 🤔. Wouldn't it be better to cast `o` to array back in the definition of `e in ( inSet )`?